tools/build.py

```diff
@@ -4,6 +4,10 @@
 import ufo2ft
 import vttLib
 import argparse
+import fontTools.ttLib
+
+
+logger = logging.getLogger(__file__)
```

Comment:
you mean `logger = logging.getLogger(__name__)`

Reply:
I did at first, then I found that it prints
```
INFO:__main__:Adding legacy kerning table to the TTF, total kerning pairs: 63335, pairs kept after flattening: 5527
```
`__file__` prints
```
INFO:tools/build.py:Adding legacy kerning table to the TTF, total kerning pairs: 63335, pairs kept after flattening: 5527
```

Reply:
yes, using `__name__` works better with importable modules rather than scripts (i.e. modules whose name is `"__main__"`.

i'm fine with leaving it `__file__`